pytaskflow/dashboard/controllers/jobs.py

```
@@ -0,0 +1,101 @@
+    @get("/details/{job_id:uuid}", name="job_details")
+    async def job_details(self, client: "Client", job_id: UUID) -> Template:
+        job = client.get_job_details(str(job_id))
+        args = []
+        kwargs = {}
+        state_data = {}
+        history = []
+        state_counts = client.get_state_counts()
+        if job:
+            try:
+                args, kwargs = client.serializer.deserialize_args(job.args, job.kwargs)
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)